samples/angular-todo/src/app/features/journey/form/form.component.ts

```diff
@@ -79,14 +94,35 @@ export class FormComponent implements OnInit {
     this.submittingForm = true;
 
     try {
+      const selectIdCallback = step?.callbacks.find(
```

Comment:
Same issue here, I'd like for this to be `selectIdPCallback`, so it properly reflects the actual name of the callback.

Reply:
done